MAINTAINERS.md

```diff
@@ -6,28 +6,27 @@ such as architecture, process, existing issues.
-| Arseniy Fokin          | @stinger112              | stinger112@gmail.com           | NodeJS library                  |
-| Artyom Bakhtin         | @bakhtin                 | a@bakhtin.net                  | Ansible, artifacts              |
-| Bulat Nasrulin         | @grimadas                | bulat@soramitsu.co.jp          | Development                     |
-| Dumitru Savva          | @x3medima17              | savva@soramitsu.co.jp          | Development                     |
-| Evgenii Mininbaev      | @l4l                     | evgenii@soramitsu.co.jp        | Security, Python library        |
-| Igor Egorov            | @igor-egorov             | igor@soramitsu.co.jp           | Development, Android library    |
-| Konstantin Munichev    | @luckychess              | konstantin@soramitsu.co.jp     | Security                        |
-| Nikita Alekseev        | @nickaleks               | alekseev@soramitsu.co.jp       | Development                     |
-| Nikolai Iushkevich     | @neewy                   | n.yushkevich@hotmail.com       | Development                     |
-| Ryu Okada              | @ryuo88                  | okada@soramitsu.co.jp          | Product vision                  |
-| Sergei Solonets        | @Solonets                | ssolonets@gmail.com            | Development                     |
-| Victor Drobny          | @victordrobny            | drobny@soramitsu.co.jp         | Development                     |
-| Vyacheslav Bikbaev     | @laSinteZ                | viacheslav@soramitsu.co.jp     | Documentation, NodeJS library   |
-| Yanno Ban              | @yannoban                | ban.yanno@nbc.org.kh           | Development                     |
+| Name                  | GitHub Id                                            | email                           | Area of expertise                                                             |
+| --------------------- | ---------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------------------- |
+| Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product vision                                                                |
+| Ryu Okada             | [@ryuo88](https://github.com/ryuo88)                 | okada@soramitsu.co.jp           | Product vision                                                                |
+| Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
+| Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Tribe Lead                                                                    |
```

Comment:
I don't think people outside the company know what we mean by tribe so it's better to have a different description here